plugins/multiplexer/ats-multiplexer.cc

```diff
@@ -77,8 +80,19 @@ TSRemapNewInstance(int argc, char **argv, void **i, char *, int)
   Instance *instance = new Instance;
 
   if (argc > 2) {
-    std::copy(argv + 2, argv + argc, std::back_inserter(instance->origins));
+    std::copy_if(argv + 2, argv + argc, std::back_inserter(instance->origins), [&](std::string s) {
+      std::regex srgx("proxy\\.config\\.multiplexer\\.skip\\_post\\_put=(\\d)");
```

Comment:
why do we need a regex to match the parameter? 
perhaps we can be a bit simpler with just a string match and expand/enhance on that if we have many more parameters later on?

Reply:
@shukitchan on it